test/resolve-test-data.js

```diff
@@ -5,6 +5,41 @@ exports.raw = [
         sys: {
           type: `Link`,
           linkType: `Space`,
+          id: `im67laxslxvo`,
+        },
+      },
+      type: `Asset`,
+      id: `4jM3XYHBu0UACAIMeAgA86`,
+      revision: 2,
+      createdAt: `2017-09-13T14:15:08.218Z`,
+      updatedAt: `2017-10-02T09:49:40.790Z`,
+    },
+    fields: {
+      title: {
+        "en-US": `AWS S3`,
+      },
+      file: {
+        "en-US": {
+          url: `//images.contentful.com/im67laxslxvo/4jM3XYHBu0UACAIMeAgA86/87f6d5a43d86495abe84e4ec5012ad66/AWS_S3__1_.png`,
```

Comment:
is this missing 'https://'?

Reply:
Thats how it comes from Contentful I think